src/worker-sdk7/sdk7-runtime.ts

```diff
@@ -63,12 +67,12 @@ export function createModuleRuntime(
 
   Object.defineProperty(runtime, 'console', {
     value: {
-      log: devtools.log.bind(devtools),
-      info: devtools.log.bind(devtools),
-      debug: devtools.log.bind(devtools),
-      trace: devtools.log.bind(devtools),
-      warning: devtools.error.bind(devtools),
-      error: devtools.error.bind(devtools)
+      log: console.log.bind(console),
```

Comment:
this will break the `Scene Logs` in the Editor
https://github.com/decentraland/unity-renderer/commit/6a6e0b365ebd7cdc86488a1b3fb455497b107690

Reply:
OH NO